### PR TITLE
add new properties

### PR DESCRIPTION
**Pull request by mimmi20**
To be able to solve detection issues found by the parser comparasation ThaDafinser/UserAgentParserComparison I like to add these 3 fields.

### CODE REVIEW

**Comment by mimmi20** [comment]
This PR is required to add these new properies to browscap/browscap.

File: src/Data/PropertyHolder.php [hunk 1]
```diff
@@ -136,6 +136,9 @@ public function getPropertyType($propertyName)
             'Crawler',
             'MasterParent',
             'LiteMode',
+            'isFake',
```
Comment (asgrim) [inline src/Data/PropertyHolder.php]: What happens when these properties don't exist in the INI file downloaded?

Comment (mimmi20) [inline src/Data/PropertyHolder.php]: Nothing. If they not in the ini files, they are ignored, like all full-file-properties when parsing a standard or lite file.

Comment (asgrim) [inline src/Data/PropertyHolder.php]: Thanks!

**Pull request merged**